Gruntfile.js

```diff
@@ -1566,7 +1566,9 @@ module.exports = function(grunt) {
 	 */
 	grunt.registerTask( 'verify:source-maps', function() {
 		const ignoredFiles = [
-			'build/wp-includes/js/dist/components.js'
+			'build/wp-includes/js/dist/components.js',
+			'build/wp-includes/js/dist/vendor/wp-polyfill-importmap.js',
+			'build/wp-includes/js/dist/vendor/wp-polyfill-importmap.min.js'
```

Comment:
@gziolo Is it normal that these are the only scripts that has `sourceMappingUrl` ?
Do you know why is this checked?

Reply:
No idea. @azaozz, do you know who could help here?

Reply:
Fixed: https://github.com/WordPress/wordpress-develop/pull/5947#discussion_r1471224745